### PR TITLE
chore: Updated the Tested up to: 5.8 - No issue found

### PR DESCRIPTION
**Pull request by nileshc-bsf**
- Tested with WordPress 5.8 RC - No issue found with it.

### CODE REVIEW

File: contacts-in-mautic.php [hunk 1]
```diff
@@ -599,4 +599,4 @@ function bsfm_connect_mautic_username_password( $data, $anonymous ) {
 	$mautic_response['body'] = $body;
 
 	return $mautic_response;
-}
\ No newline at end of file
+}
```
Comment (patilvikasj) [inline contacts-in-mautic.php:602]: @nileshc-bsf Discard this change as this will prevent from releasing readme update.

**Review by patilvikasj — CHANGES_REQUESTED** [review]
Added a comment

File: contacts-in-mautic.php (hunk 1 above)
Comment (nileshc-bsf) [reply contacts-in-mautic.php:602]: Okay, but here pipeline will fail, due to PHPCS.

Comment (patilvikasj) [reply contacts-in-mautic.php:602]: @nileshc-bsf  Do this change in the next bugfix / minor update. Let the PHPCS pipeline fail for this PR.

Comment (nileshc-bsf) [reply contacts-in-mautic.php]: @patilvikasj - Removed the changes.

**Review by patilvikasj — APPROVED** [review]

**Pull request merged**